### PR TITLE
DOCS-1657:  R2D2 Update add component and service instructions

### PR DESCRIPTION
**Pull request by sguequierre**
* Update all instructions for R2D2

This is purposefully missing tutorials

* **Remember to do a final sweep after merging in new changes to check for missed spots in between**

### CODE REVIEW

File: docs/components/power-sensor/fake.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ Configure a `fake` power sensor to integrate into your machine:
...
 {{% tab name="Config Builder" %}}
 
-Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com).
-Click on the **Components** subtab and click **Create component** in the lower-left corner.
-Select the type `power_sensor`, then select the `fake` model.
-Name your sensor, and click **Create**.
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Select the `power_sensor` type, then select the `fake` model.
+Enter a name for your sensor and click **Create**.
```
Comment (andf-viam) [inline docs/components/power-sensor/fake.md:25]: supernit: let's match the other pages where the type of sensor is also explicitly referenced:

```suggestion
Enter a name for your power sensor and click **Create**.
```

and on other power sensor pages.

File: docs/data/capture.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ However, if you want to manage your machine's captured data yourself, you can en
 
 To capture data from one or more machines, you must first add the [data management service](../):
 
-1. From your machine's **Config** tab, navigate to the **Services** subtab.
-2. Click **Create service** in the lower-left corner of the page.
-   Choose `Data Management` as the type and specify a name for your data management service, for example `data-manager`.
-3. Click **Create**.
-4. On the panel that appears, you can manage the capturing and syncing functions and specify the **directory**, the sync **interval** and any **tags** to apply to captured data.
+1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
```
Comment (andf-viam) [inline docs/data/capture.md:37]: Nit: we tend to use `on your | the page` over `of`:

```suggestion
1. Navigate to the **CONFIGURE** tab on your machine's page in [the Viam app](https://app.viam.com).
```

**Review by andf-viam — APPROVED** [review]
(sorry clicked too soon earlier!)

LGTM, thanks so much! Just a thought on `on your ... page` over `from your ... page` if you agree.

Thanks!

File: docs/data/capture.md (hunk 2 above)
Comment (sguequierre) [reply docs/data/capture.md:37]: I like of... sorry 😆

Comment (sguequierre) [reply docs/data/capture.md:37]: I think on sounds good too but I don't think it absolutely needs to be changed

File: docs/data/capture.md [hunk 3]
```diff
@@ -34,11 +34,11 @@ However, if you want to manage your machine's captured data yourself, you can en
 
 To capture data from one or more machines, you must first add the [data management service](../):
 
-1. From your machine's **Config** tab, navigate to the **Services** subtab.
-2. Click **Create service** in the lower-left corner of the page.
-   Choose `Data Management` as the type and specify a name for your data management service, for example `data-manager`.
-3. Click **Create**.
-4. On the panel that appears, you can manage the capturing and syncing functions and specify the **directory**, the sync **interval** and any **tags** to apply to captured data.
+1. Navigate to the **CONFIGURE** tab on your machine's page in [the Viam app](https://app.viam.com).
```
Comment (sguequierre) [inline docs/data/capture.md:37]:
```suggestion
1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
```

**Pull request merged**